### PR TITLE
Added Course POJO

### PR DESCRIPTION
**Pull request by MohammdWakeel**
Created POJO class of Course

### CODE REVIEW

File: src/main/java/com/revature/studentmanagementsystem/model/Course.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Data
+@Entity
+@Table
+public class Course {
+
```
Comment (sargarakshay) [inline src/main/java/com/revature/studentmanagementsystem/model/Course.java:13]: Try to avoid the unnecessary spaces.

**Review by lalitghongade — APPROVED** [review]
looks good,done with task SMS-7 :)

**Review by sumansuruchi — APPROVED** [review]
Good job

**Review by sargarakshay — APPROVED** [review]
Looks Good To Me :)

File: src/main/java/com/revature/studentmanagementsystem/model/Course.java (hunk 1 above)
Comment (MohammdWakeel) [reply src/main/java/com/revature/studentmanagementsystem/model/Course.java:13]: Ok I will do that

Comment (MohammdWakeel) [reply src/main/java/com/revature/studentmanagementsystem/model/Course.java:13]: I have resolved

**Pull request merged**